static/app/utils/replays/hooks/useReplayData.tsx

```diff
@@ -230,12 +233,23 @@ function useReplayData({
   }, [orgSlug, replayId, projectSlug, queryClient]);
 
   return useMemo(() => {
+    // This hook can enter a state where `fetching` below is false
+    // before it is entirely ready (i.e. it has not fetched
+    // attachemnts yet). This can cause downstream components to
+    // think it is no longer fetching and will display an error
+    // because there are no attachments. The below will require
+    // that we have attempted to fetch an attachment once (or if it
+    // errors) before we toggle fetching state to false.
+    hasFetchedAttachments.current =
+      hasFetchedAttachments.current || isFetchingAttachments;
```

Comment:
please double check my logic, but i'm pretty sure this is right and pretty sure we _always_ want to make at least 1 request for attachments or something is wrong.